src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/Models/SemanticTokens.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp.Models
+{
+    /// <summary>
+    /// Class representing response to semantic tokens messages.
+    /// </summary>
+    [DataContract]
```

Comment:
I assume this type was copy and pasted from the Language Server platform right? We don't need all the DataContract pieces on our side. For instance, here's how O# represents it: https://github.com/OmniSharp/csharp-language-server-protocol/blob/d82ff8211d32a18b6df8aeb4b56fa5635b45188f/src/Protocol/Models/Proposals/SemanticTokens.cs

In ours we wouldn't want it obsolete but we don't need all the data contract pieces, ditto in the other contract types

Reply:
Actually, I just noticed this was in the LanguageServerClient.Razor/HtmlCSharp library. Why do we need this at all?